### PR TITLE
Add a couple test sprites for Game Lab Jr

### PR DESCRIPTION
**Pull request by balderdash**
Add default animations named "dog" and "cat" to the gamelab jr project template.
image

### CODE REVIEW

**Comment by caleybrock** [comment]
screen shot 2018-02-19 at 3 41 13 pm
There are a couple of dogs in a matching cat style if you care. But the corgi is fun too.

File: dashboard/config/scripts/levels/New Game Lab Jr Project.level [hunk 1]
```diff
@@ -214,11 +214,17 @@
...
+    "start_animations": "{\r\n  \"orderedKeys\": [\r\n    \"9f2f01be-42ae-4242-9b47-3476cf8f1f5c\",\r\n    \"0dba258d-a2fa-471c-92d6-42f19684746e\"\r\n  ],\r\n  \"propsByKey\": {\r\n    \"9f2f01be-42ae-4242-9b47-3476cf8f1f5c\": {\r\n      \"name\": \"cat\",\r\n      \"sourceUrl\": \"http://localhost-studio.code.org:3000/v3/animations/p7sb0a3gMNYkzRBreemCkA/9f2f01be-42ae-4242-9b47-3476cf8f1f5c.png?version=_naL8ux9jreFvP_mwIcx4QZUf9QapDlj\",\r\n      \"frameSize\": {\r\n        \"x\": 64,\r\n        \"y\": 100\r\n      },\r\n      \"frameCount\": 1,\r\n      \"looping\": true,\r\n      \"frameDelay\": 12,\r\n      \"version\": \"_naL8ux9jreFvP_mwIcx4QZUf9QapDlj\"\r\n    },\r\n    \"0dba258d-a2fa-471c-92d6-42f19684746e\": {\r\n      \"name\": \"dog\",\r\n      \"sourceUrl\": \"http://localhost-studio.code.org:3000/v3/animations/p7sb0a3gMNYkzRBreemCkA/0dba258d-a2fa-471c-92d6-42f19684746e.png?version=psbxMDrISyQzlR0WaHxCcihPItagUuq2\",\r\n      \"frameSize\": {\r\n        \"x\": 100,\r\n        \"y\": 92\r\n      },\r\n      \"frameCount\": 1,\r\n      \"looping\": true,\r\n      \"frameDelay\": 12,\r\n      \"version\": \"psbxMDrISyQzlR0WaHxCcihPItagUuq2\"\r\n    }\r\n  }\r\n}"
```
Comment (caleybrock) [inline dashboard/config/scripts/levels/New Game Lab Jr Project.level]: you probably want to use production animation URLs instead of localhost ones.

Comment (balderdash) [reply dashboard/config/scripts/levels/New Game Lab Jr Project.level]: Thanks for catching that.

**Comment by balderdash** [comment]
I think I'll stick with the corgi for now.

**Pull request merged**